src/include/concurrency/transaction_manager.h

```diff
@@ -150,24 +150,11 @@ class TransactionManager {
     return EpochManagerFactory::GetInstance().GetExpiredCid();
   }
 
-  void SetDirtyRange(std::pair<cid_t, cid_t> dirty_range) {
-    this->dirty_range_ = dirty_range;
-  }
-
   IsolationLevelType GetIsolationLevel() {
     return isolation_level_;
   }
 
  protected:
```

Comment:
This "dirty range" is used by the WBL. As the logging components are under refactoring, I removed this code for now. \cc @PauloAmora